backend/controllers/userController.js

```diff
@@ -7,11 +7,11 @@ const User = require("../model/user.model")
 
 const registerUser = async (req, res, next) => {
   try {
-    const { name, email, password, confirmpwd, phone } = req.body
+    let { name, email, password, confirmpwd, phone } = req.body
```

Comment:
only email value is changing!! other values can be const

Reply:
done